README.md

```diff
@@ -102,15 +120,14 @@ func main() {
 ```
 
 
-Acceptance Tests
-----------------
+## Acceptance Tests
 
 If you want to run tests against the actual Packet API, you must set envvar `PACKET_TEST_ACTUAL_API` to non-empty string for the `go test`. The device tests wait for the device creation, so it's best to run a few in parallel.
 
 To run a particular test, you can do
 
 ```
-$ PACKNGO_TEST_ACTUAL_API=1 go test -v -run=TestAccDeviceBasic
+$ PACKNGO_TEST_ACTUAL_API=1 go test -v -run=TestAccDeviceBasic --timeout=2h
 ```
```

Comment:
You can also mention that if it fails because lack of devices in a facility, it's possible to set `PACKNGO_TEST_FACILITY` for the tests to go to a different fac